src/modules/commander/HealthAndArmingChecks/checks/escCheck.cpp

```diff
@@ -77,7 +77,7 @@ void EscChecks::checkAndReport(const Context &context, Report &reporter)
 
 	esc_status_s esc_status;
 
-	if (_esc_status_sub.copy(&esc_status) && now - esc_status.timestamp < esc_telemetry_timeout) {
+	if (_esc_status_sub.copy(&esc_status) && now < esc_telemetry_timeout + esc_status.timestamp) {
```

Comment:
```suggestion
	if (_esc_status_sub.copy(&esc_status) && (now < esc_status.timestamp + esc_telemetry_timeout)) {
```
Just a nitpick for readability while we're changing it already.